### PR TITLE
Allow the user to reset input actions for all controller types.

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Users can now reset the input actions for all controller types. The UI has also been updated to streamline this new functionality

Old UI
image

New UI
Inspectornew2

## Changes
- Fixes: #7521

### CODE REVIEW

File: Assets/MRTK/Core/Inspectors/Profiles/MixedRealityControllerMappingProfileInspector.cs [hunk 1]
```diff
@@ -325,13 +322,25 @@ private void RenderControllerList(SerializedProperty controllerList)
...
                             {
                                 ControllerMappingSignature buttonSignature = new ControllerMappingSignature(supportedControllerType, handedness);
                                 ControllerPopupWindow.Show(controllerMapping, interactionsProperty, handedness, controllersAffectedByMappingSignatures[buttonSignature]);
                             }
+                            if (GUILayout.Button(EditorGUIUtility.IconContent("_Menu"), new GUIStyle("iconButton")))
+                            {
+                                // create the menu and add items to it
+                                GenericMenu menu = new GenericMenu();
+
+                                //Caching the index of this controller mapping for the anonymous fucntion
```
Comment (keveleigh) [inline Assets/MRTK/Core/Inspectors/Profiles/MixedRealityControllerMappingProfileInspector.cs:338]:
```suggestion
                                // Caching the index of this controller mapping for the anonymous function
```

File: Assets/MRTK/Core/Inspectors/ControllerPopupWindow.cs [hunk 2]
```diff
@@ -254,6 +254,14 @@ public static void Show(MixedRealityControllerMapping controllerMapping, Seriali
             defaultFieldWidth = EditorGUIUtility.fieldWidth;
         }
 
+        public static void RepaintWindow()
```
Comment (keveleigh) [inline Assets/MRTK/Core/Inspectors/ControllerPopupWindow.cs:257]: nit: missing summary (though, it looks like this entire class is...)

**Review by keveleigh — APPROVED** [review]

**Comment by keveleigh** [comment]
This is awesome!!

File: Assets/MRTK/Core/Inspectors/ControllerPopupWindow.cs (hunk 2 above)
Comment (RogPodge) [reply Assets/MRTK/Core/Inspectors/ControllerPopupWindow.cs:257]: I'll do some maintenance :)

**Pull request merged**